### ISSUE TITLE
Make the images in the catalogue and portfolio indices clickable

### ISSUE THREAD

**Issue by RyotaMitaraiWeb** [RyotaMitaraiWeb/studio-dada-website#66]
Clicking those images should lead the user to the respective page, similarly to how the buttons work.

To prevent redundancy for screenreaders, those links should have a tabindex of -1 and be hidden from them.
